cmd/metal-api/internal/service/tenant-service.go

```diff
@@ -72,6 +73,17 @@ func (r *tenantResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", []v1.TenantResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.POST("/{id}/history").
+		To(viewer(r.getTenantHistory)).
+		Operation("getTenantHistory").
+		Doc("get tenant with this id at the given timestamp").
+		Param(ws.PathParameter("id", "identifier of the tenant").DataType("string")).
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Reads(internalV1.TenantGetHistoryRequest{}).
```

Comment:
@Gerrit91  Would it be possible to just reuse [the original](https://github.com/metal-stack/masterdata-api/blob/master/api/v1/tenant.pb.go#L300), or would that be a bad pratice?

Reply:
We should not mix the GRPC APIs with the REST APIs.

Reply:
What you could do is adding the request to the REST definition in the masterdata-api: https://github.com/metal-stack/masterdata-api/blob/master/api/rest/v1/tenant.go